share/isso.conf

```diff
@@ -188,9 +188,10 @@ require-email = false
 # Customize markup and sanitized HTML. Currently, only Markdown (via Misaka) is
 # supported, but new languages are relatively easy to add.
 
-# Misaka-specific Markdown extensions, all flags starting with EXT_ can be used
-# there, separated by comma.
-options = strikethrough, autolink, fenced_code, no_intra_emphasis
+# Misaka-specific Markdown extensions, all extensions can be used here,
+# separated by comma, either by their name or by EXT_<extension>.
+# Careful: Misaka 1.0 used "snake_case", but 2.0 needs "dashed-case"!
```

Comment:
We've been on misaka 2.0 since 1.5, do we really need to mention misaka 1.0?

Reply:
I was directing that comment towards people upgrading from older versions.